packages/cli/src/metrics/__tests__/license-metrics.service.test.ts

```diff
@@ -60,6 +61,10 @@ describe('LicenseMetricsService', () => {
 				{ name: 'totalUsers', value: mockRenewalMetrics.totalUsers },
 				{ name: 'totalCredentials', value: mockRenewalMetrics.totalCredentials },
 				{ name: 'productionExecutions', value: mockRenewalMetrics.productionExecutions },
+				{
+					name: 'billableProductionExecutions',
```

Comment:
It's been called `productionRootExecutions` in the licenses. Do you think we should change it?
```suggestion
					name: 'productionRootExecutions',
```